app/views/donors/_edit.html.erb

```diff
@@ -0,0 +1,44 @@
+<%= form_for(@donor) do |f| %>
+
+<div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
```

Comment:
Since this covers the whole 12 columns, we don't need to specify any grid unit classes.